google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java

```diff
@@ -227,8 +230,11 @@ public CloudSdk createSdk(
         .appCommandCredentialFile(credentialsPath.toFile())
         .appCommandMetricsEnvironment(pluginInfoService.getExternalPluginName())
         .appCommandMetricsEnvironmentVersion(pluginInfoService.getPluginVersion())
-        .appCommandOutputFormat("json")
-        .build();
+        .appCommandOutputFormat("json");
+
+    getProjectJavaSdk(project).ifPresent(sdkBuilder::javaSdkPath);
```

Comment:
This also doesn't fit well with the current CloudSdk architecture, where e.g. [CloudSdkDevAppEngineDevServer](https://github.com/GoogleCloudPlatform/appengine-plugins-core/blob/master/src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer.java#L105) translates a configuration into flags and then the [CloudSdk](https://github.com/GoogleCloudPlatform/appengine-plugins-core/blob/master/src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java#L251) object executes it.

I think the change from specifying jdkHome from CloudSdk config to command config needs a lot more thinking, and I'm not comfortable doing that in the time frame required to fix the Standard staging issue. I could go ahead and hack something to make the command config way fit it with the current CloudSdk model, but I don't think that would be the right approach.